sdk/src/cameras/ad-96tof1-ebz/calibration_96tof1.h

```diff
@@ -113,10 +116,17 @@ class Calibration96Tof1 {
                                     int16_t maxPixelValue, int range);
     void buildGeometryCalibrationCache(const std::vector<float> &cameraMatrix,
                                        unsigned int width, unsigned int height);
+    void
+    buildDistortionCorrectionCache(const std::vector<float> &cameraMatrix,
+                                   const std::vector<float> &distortionCoeffs,
+                                   unsigned int width, unsigned int height);
 
   private:
     std::unordered_map<float, packet_struct> m_calibration_map;
+    double *m_intrinsics;
+    double *m_distCoeffs;
```

Comment:
initialize this with nullptr in constructor's initialiazation list. Otherwise if camera gets destroyed but m_distCoeffs didn't get an allocated memory, in destructor it will end up freeing an invalid pointer.

Reply:
Same things for m_instrinsics